language/de-DE/joomla.ini

```
@@ -100,6 +100,7 @@ JOK="OK"
 JOFF="Aus"
 JOFFLINE_MESSAGE="Diese Website ist auf Grund von Wartungsarbeiten nicht erreichbar.<br />Bitte später wiederkommen."
 JON="An"
+JOPEN="Offen"
```

```suggestion
JOPEN="Öffnen"
```

With regard to JCLOSE, this does not mean Open but Opening :)

Right? So a change in /administrator/langauge/de-DE/joomla.ini would be nice.